vercel.json

```diff
@@ -918,6 +918,11 @@
       "source": "/docs/kratos/mfa/requesting-2fa",
       "destination": "https://www.ory.sh/docs/kratos/mfa/step-up-authentication",
       "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/docker",
```

Comment:
@tomekpapiernik Did you mean to reference `/docs/kratos/guides/docker` on this line?

Reply:
No, this is correct. It makes it so that anyone who lands on the doc indicated in `source` is redirected to the address defined in `destination`.

Reply:
Ok, got it. That's a handy feature.